Backend/dev/test-tool/load-test-service/locust_runner.py

```
@@ -0,0 +1,145 @@
+    abort = threading.Event()
+    t0 = time.time()
+
+    def _reader():
+        try:
+            for raw in proc.stderr:
+                line = raw.rstrip()
+                if line.startswith('LT_EVT:'):
+                    try:
+                        eq.put(json.loads(line[7:]))
+                    except Exception:
+                        pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)